packages/core/src/Stores/client-store.js

```diff
@@ -965,39 +965,54 @@ export default class ClientStore extends BaseStore {
         return this.isBotAllowed();
     }
 
-    getIsMarketTypeMatching = (account, market_type) =>
-        market_type === 'synthetic'
-            ? account.market_type === market_type || account.market_type === 'gaming'
-            : account.market_type === 'financial';
+    getIsMarketTypeMatching = (account, market_type) => {
+        if (market_type === 'synthetic') {
+            return account.market_type === market_type || account.market_type === 'gaming';
+        } else if (market_type === 'all') {
+            return account.market_type === 'all';
+        }
+        return account.market_type === 'financial';
+    };
 
     isEligibleForMoreDemoMt5Svg(market_type) {
+        const is_synthetic = market_type === 'synthetic';
+        const is_all = market_type === 'all';
+        const is_swap_free_account = is_all ? 'all' : 'financial';
```

Comment:
```suggestion
        const is_swap_free_account = market_type === 'all' ? 'all' : 'financial';
```
This looks like it can be in one declaration

Reply:
ok